### PR TITLE
Kmeans fix

### LINKED ISSUE #44: Example kmeans.py raises an error

### PR DESCRIPTION
**Pull request by javicid**
Closes #44

### CODE REVIEW

File: dislib/cluster/kmeans/base.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def fit_predict(self, dataset):
         for subset in dataset:
             labels.append(_get_label(subset))
 
-        return np.array(compss_wait_on(labels))
+        return np.concatenate(compss_wait_on(labels))
```
Comment (kafkasl) [inline dislib/cluster/kmeans/base.py]: is more efficient np.concatenate than np.array? asking out of curiosity to learn.

File: dislib/cluster/kmeans/base.py [hunk 2]
```diff
@@ -122,9 +122,10 @@ def predict(self, x):
         """
         labels = []
 
-        for x in x:
-            dist = np.linalg.norm(x - self.centers, axis=1)
+        for sample in x:
+            dist = np.linalg.norm(sample - self.centers, axis=1)
```
Comment (kafkasl) [inline dislib/cluster/kmeans/base.py]: big fan of having explanatory names for vars :+1:

File: tests/tests.py [hunk 3]
```diff
@@ -27,6 +27,30 @@ def test_cascadecsvm(self):
         from dislib.classification import CascadeSVM
 
 
+class KMeansTest(unittest.TestCase):
+
+    def test_fit_predict(self):
+        import numpy as np
+        from sklearn.datasets import make_blobs
+        from dislib.data import load_data
+        from dislib.cluster import KMeans
```
Comment (kafkasl) [inline tests/tests.py]: put the imports on top

File: dislib/cluster/kmeans/base.py (hunk 1 above)
Comment (javicid) [reply dislib/cluster/kmeans/base.py]: No, the issue was that a concatenation is necessary here.

File: tests/tests.py (hunk 3 above)
Comment (javicid) [reply tests/tests.py]: Done

**Pull request merged**